modules/aws_dlm_lifecycle_policy/default/0.1/facets.yaml

```
@@ -4,8 +4,126 @@ version: '0.1'
+              type: object
+              title: Tags to Add
+              description: Additional tags to apply to snapshots
+              x-ui-yaml-editor: true
+          required: ["copy_tags", "create_rule", "retain_rule", "tags_to_add"]
+    target_tags:
+      type: object
+      title: Target Tags
+      description: Tags used to identify volumes for snapshot policy
+      x-ui-yaml-editor: true
+  required:
+    - schedules
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)